src/coprocessor/endpoint.rs

```diff
@@ -605,15 +607,63 @@ mod tests {
         ) -> UnaryFixture {
             UnaryFixture {
                 handle_duration_millis,
+                yieldable: false,
+                result: Some(result),
+            }
+        }
+
+        pub fn new_with_duration_yieldable(
+            result: Result<coppb::Response>,
+            handle_duration_millis: u64,
+        ) -> UnaryFixture {
+            UnaryFixture {
+                handle_duration_millis,
+                yieldable: true,
                 result: Some(result),
             }
         }
     }
 
+    async fn yield_now() {
```

Comment:
You can use `yatp::task::future::reschedule` instead of rewriting a `yield_now` :)